.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: pip install poetry==1.4.2
       - name: Install dependencies
         run: poetry install --all-extras
-      - name: Lint with pylint
-        run: poetry run pylint pandasai examples
+      - name: Lint with ruff
+        run: bash scripts/lint.sh
```

Comment:
Maybe CI is failing because the script isn't running in the `poetry` environment. Why not just `poetry run rust check pandasai examples`?

Reply:
@alejohz did you have a chance to give it a try?